extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -183,7 +189,7 @@ List<MessageBundleBuildItem> processBundles(BeanArchiveIndexBuildItem beanArchiv
                     // Find localizations for each interface
                     String defaultLocale = getDefaultLocale(bundleAnnotation, locales);
                     List<ClassInfo> localized = new ArrayList<>();
-                    for (ClassInfo implementor : index.getKnownDirectImplementors(bundleClass.name())) {
+                    for (ClassInfo implementor : index.getKnownDirectSubinterfaces(bundleClass.name())) {
```

Comment:
I'm not sure if this is intentional, but the replacement for deprecated `getKnownDirectImplementors()` is `getKnownDirectImplementations()`. On the other hand, `getKnownDirectSubinterfaces()` only returns interfaces, not implementations of the given interface. 

Probably intentional, though, given the line below. (Which is no longer necessary :-) )

Reply:
The entire point of `getKnownDirectImplementations()` _is_ to have a different behavior compared to `getKnownDirectImplementors()` :-) Subinterfaces are not implementations of the interface.

Reply:
> Subinterfaces are not implementations of the interface.

That's why the original method used the term "implementors" :D